main/java/BitsNPicas/src/com/kreative/bitsnpicas/main/ViewFont.java

```diff
@@ -48,6 +48,17 @@ else if (bm.getName(BitmapFont.NAME_FAMILY) != null) {
 			setTitle(bm.getName(BitmapFont.NAME_FAMILY));
 		}
 
+		bm.addFontListener(new FontListener() {
+			public void metricsChanged(com.kreative.bitsnpicas.Font<?> font) {
+				if (textPanel != null) textPanel.repaint();
+				if (alphaPanel != null) alphaPanel.repaint();
+			}
+			public void glyphsChanged(com.kreative.bitsnpicas.Font<?> font) {
```

Comment:
The full name is necessary to distinguish from `java.awt.Font`. Not sure if this is preferred over expanding the `import java.awt.*` wildcard.